.drone.yml

```diff
@@ -191,6 +191,104 @@ volumes:
   host:
     path: /var/run/docker.sock
 
+---
+kind: pipeline
+name: default-s390x
+
+platform:
+  os: linux
+  arch: amd64
```

Comment:
Should this be `s390x`?

Reply:
It's a workaround because Drone does not support s390x directly.

The more detailed explanation is available here: https://confluence.suse.com/pages/viewpage.action?pageId=925925417#HowRancherproductsarebuiltfors390architecture(Z)-Specialneedsfors390x